src/storage/store.js

```diff
@@ -318,7 +319,7 @@ export default class Store extends EventTarget {
     }
 
     // Regenerate name to encourage users to change it.
-    if (!this.state.activity.hasChangedName) {
+    if (!this.state.activity.hasChangedName || isLockedDownDemoRoom()) {
```

Comment:
I don't think we want to update the display name. We are not really showing this anywhere as we have removed the nametag, the logs and the people panel so we should be good with resetting this whole file.

Reply:
Agreed!